### PR TITLE
Tyler Rosenberg - Emergency Compliment HW

### PR DESCRIPTION
**Pull request by TyRyanR**
Comfort: 3.8
Completeness: 4

Problems on this HW:

****\* I do not understand how to use YIELD. Can we go over this in class? Thanks

1) I do not understand why we needed to create two separate view ERB files, one being the compliments.erb and one being the layout.erb. I did everything in one ERB file called compliments.erb

2) Not sure if I did the view file correctly. My solution was to put the boilerplate HTML file in there, and use the instance variable @background_color in the server.rb file, in the compliments.erb file.
- In other words... Not sure if using an HTML skeleton with the head, body, etc is the correct format to use in an ERB file.

### CODE REVIEW

**Comment by nayana487** [comment]
You probably got this now, but basically layout is what appears on all templates and yield provides the html from the individual templates.
